package-lock.json

```diff
@@ -0,0 +1,1669 @@
+{
```

Comment:
We already have a yarn.lock file. We have no need for this.
Also, it is a bad practice to keep 2 lockfiles.

So, please delete this one and install dependencies using yarn.

Reply:
sure i will